tests/unit/test_bigquery_connection_manager.py

```diff
@@ -65,7 +65,9 @@ def test_is_retryable(self):
         service_unavailable_error = exceptions.ServiceUnavailable("service is unavailable")
 
         self.assertTrue(_is_retryable(internal_server_error))
-        self.assertTrue(_is_retryable(bad_request_error))
+        self.assertFalse(
```

Comment:
It's worth keeping this, but flipping it, to be clear that this was once supported, and no longer is supported.